### DISCUSSION TITLE
不知道是否侵权

### DISCUSSION THREAD

**Discussion by rstyin** [Meloong-Git/PCL#1986] [提问] [answered]
我们服务器稍微改了下pcl把里面的很多功能改没了（怕装扶助mod开挂），但不收费，举一些例子，我们把添加材质包文件加密了，把pcl处改版本和登陆号全改没了，不知道算不算侵权，如果算的话我们赶紧改

**Comment by WForst-Breeze** [comment]
没太看懂是什么意思🤔一般来讲只要不二次分发快照版本或使用 PCL2 盈利就不会构成侵权。正常的客户端自定义也不构成侵权

**Reply by rstyin** [reply]
我们是启动器不要钱但服务器要钱

**Comment by LTCatt** [comment, marked as answer]
单纯整合包/服务器修改一下自用倒是无所谓，不介意 =。=

**Reply by rstyin** [reply]
好的
